### PR TITLE
`aspectRatio` on summary and fixed styles.

### PR DESCRIPTION
**Pull request by iaronaraujo**
#### What problem is this solving?

As the title says
Reference: https://www.figma.com/file/DnpQxbX4N5CsYtr0xdiRRn/02.-Store---Price?node-id=52%3A172

#### How should this be manually tested?

[Workspace](https://iaronaraujo--storecomponents.myvtex.com/)

### CODE REVIEW

File: styles/css/vtex.product-summary.css [hunk 1]
```diff
@@ -36,3 +36,7 @@
 .imageContainer {
   text-align: center;
 }
+
+.image {
+  border-radius: 4px;
```
Comment (Klynger) [inline styles/css/vtex.product-summary.css:41]: I don't know if my suggestion is better, but I think that using `rem` would make it closer with the [tachyons token](https://twitter.com/RafaelKlynger/status/1246931223915749382)
```suggestion
  border-radius: 0.25rem;
```

File: styles/css/vtex.product-price.css [hunk 2]
```diff
@@ -37,7 +38,8 @@
 }
 
 .savings-discount--summary {
```
Comment (Klynger) [inline styles/css/vtex.product-price.css:40]: Is this class being applied in any element?

Comment (Klynger) [reply styles/css/vtex.product-price.css:40]: Found it, sry

Comment (iaronaraujo) [reply styles/css/vtex.product-price.css:40]: Yes! 
image

File: styles/css/vtex.product-summary.css (hunk 1 above)
Comment (iaronaraujo) [reply styles/css/vtex.product-summary.css:41]: But `rem` is relative to the screen right? I don't think that we want to change the border if the screen changes, but I can check this out

**Comment by iaronaraujo** [comment]
I've reset the workspace and forgot to link the theme again, sorry!

File: styles/css/vtex.product-summary.css (hunk 1 above)
Comment (iaronaraujo) [reply styles/css/vtex.product-summary.css]: Done!

**Pull request merged**